src/os/bluestore/BlueStore.cc

```diff
@@ -18512,33 +18512,19 @@ int BlueStore::copy_allocator(Allocator* src_alloc, Allocator* dest_alloc, uint6
 }
 
 //-----------------------------------------------------------------------------------
-size_t calc_allocator_image_header_size()
+size_t calc_allocator_image_field_size(bool trailer)
```

Comment:
The simplification of calc_allocator_image_header/tailer_size is very good, 
but switch to calc_allocator_image_field_size(bool trailer) makes it less clear.